backend-shared/divesummary.cpp

```diff
@@ -95,6 +95,13 @@ void diveSummary::loopDives(timestamp_t primaryStart, timestamp_t secondaryStart
 
 void diveSummary::calculateDive(int inx, struct dive *dive)
 {
+	// Check for dive validity
+	// Swimmingpool dives (4 meters) are not counted, nor
+	// are very short dives (5 minutes)
+	if (dive->maxdepth.mm < 4000 ||
+		dive->duration.seconds < 5 * 60)
+		return;
```

Comment:
I don't think arbitrary thresholds like this are a good idea.
THIS is the real reason to have an invalid dive flag.
If the user wants to exclude certain dives, we should give them the option to do so.
But we should absolutely not create a "summary" that arbitrary excludes dives from the summary.
So NAK on this change.